include/dd/Package.hpp

```diff
@@ -557,7 +571,7 @@ namespace dd {
                 // Collecting garbage in the complex numbers table requires collecting the node tables as well
                 force = true;
             }
-            auto mCollect = mUniqueTable.garbageCollect(force);
+            auto mCollect = mUniqueTable.garbageCollect(force) + dUniqueTable.garbageCollect(force);
```

Comment:
aren't these separate node types and, hence, should also be tracked in different variables?

Reply:
I added an extra variable with `dCollect `. Note that I collect the same caches when `mCollect > 0` or `dCollect > 0`.

I therefore use `if (mCollect > 0 || dCollect > 0) {
                matrixAdd.clear();`